docs/get-started/python.md

```diff
@@ -0,0 +1,127 @@
+# Getting started with CDK8s for Python
+In this guide, we'll walk you through the most essential concepts you need to know in order to build a Kubernetes application using CDK8s for Python.
+
+## Prerequisites
+1. CDK8s for Python supports Python versions 3.7+. Check your Python version:
+```bash
+python3 --version
+```
+
+## Setting up an environment
+### Initial setup
+To get setup, follow these general steps:
+
+1. To begin, you’ll need a development environment with pipenv 2018.11.26+ installed. If you’re not familiar with this setup, use these instructions to [setup pipenv](https://pipenv.pypa.io/en/latest/install/).
+
+### Install the CLI
+To initialize a CDK8s project and auto-generate Kubernetes manifests based on our code, we need the CDK8s CLI:
+
+1. Run the following command to install the CLI using [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). For more installation methods, see [Install the CDK8s CLI](cli/installation.md).
+```console
+npm install -g cdk8s-cli
+```
+
+### Create a project
+Next, we’ll initialize a project to create the directory structure, install the CDK8s and CDK8s+ packages and all of the dependencies required to run an application using the [init command](https://cdk8s.io/docs/latest/cli/init/). 
+
+1. In a terminal window, run the `init` command in an empty directory: 
```

Comment:
```suggestion
1. In a terminal window, run the `cdk8s init` command in an empty directory: 
```

Reply:
Since we already linked to/indicated we're using the init command, changed to "run the following command"